### PR TITLE
Fix issue 409: Clarify instructions in CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by sgilda**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -34,22 +34,36 @@ Here are the steps in detail:
...
 
-        git push origin HEAD            
+        git commit -m 'Description of change...'
+
+10. Push your local topic branch to your github forked repository. This will create a branch on your Git fork repository with the same name as your local topic branch name. 
 
-9. Browse to the <topic-branch-name> branch on your forked Git repository and [open a Pull Request](http://help.github.com/send-pull-requests/). Give it a clear title and description.
+        git push origin HEAD            
+   _Note: The above command assumes your remote repository is named 'origin'. You can verify your forked remote repository name using the command `git remote -v`_.
+11. Browse to the <topic-branch-name> branch on your forked Git repository and [open a Pull Request](http://help.github.com/send-pull-requests/). Give it a clear title and description.
```
Comment (LightGuard) [inline CONTRIBUTING.md:66]: git push origin HEAD ??

Shouldn't HEAD => the name of the branch created in step5? saying HEAD just seems to get into more git ideas than are needed.

Comment (sgilda) [inline CONTRIBUTING.md:66]: @LightGuard : When I test changes on my own branch, I always use HEAD and it creates a branch on my remote repository with the same name as my local branch.  I looked back through chat logs and found I got this command from @pmuir originally.

I actually didn't change this line. It just got moved down from line 50 above when I inserted more information about git add, git status, and commits. :-)

Do you not need the HEAD?

Comment (LightGuard) [inline CONTRIBUTING.md:66]: You have to give it some name. If HEAD creates the same name as the local
branch on the remote side, great. More than one way to do the same thing.

On Mon, Mar 4, 2013 at 1:57 PM, sgilda notifications@github.com wrote:

> In CONTRIBUTING.md:
> 
> > -9. Browse to the <topic-branch-name> branch on your forked Git repository and [open a Pull Request](http://help.github.com/send-pull-requests/). Give it a clear title and description.
> > -        git push origin HEAD
> > -   _Note: The above command assumes your remote repository is named 'origin'. You can verify your forked remote repository name using the command `git remote -v`_.
> >   +11. Browse to the <topic-branch-name> branch on your forked Git repository and [open a Pull Request](http://help.github.com/send-pull-requests/). Give it a clear title and description.
> 
> @LightGuard https://github.com/LightGuard : When I test changes on my
> own branch, I always use HEAD and it creates a branch on my remote
> repository with the same name as my local branch. I looked back through
> chat logs and found I got this command from @pmuirhttps://github.com/pmuiroriginally.
> 
> I actually didn't change this line. It just got moved down from line 50
> above when I inserted more information about git add, git status, and
> commits. :-)
> 
> Do you not need the HEAD?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/jboss-jdf/jboss-as-quickstart/pull/462/files#r3232192
> .

## 

Jason Porter
http://en.gravatar.com/lightguardjp

Comment (sgilda) [inline CONTRIBUTING.md:66]: Are you good with this then? Can I merge?

Comment (LightGuard) [inline CONTRIBUTING.md:66]: Yep

On Mon, Mar 4, 2013 at 2:19 PM, sgilda notifications@github.com wrote:

> In CONTRIBUTING.md:
> 
> > -9. Browse to the <topic-branch-name> branch on your forked Git repository and [open a Pull Request](http://help.github.com/send-pull-requests/). Give it a clear title and description.
> > -        git push origin HEAD
> > -   _Note: The above command assumes your remote repository is named 'origin'. You can verify your forked remote repository name using the command `git remote -v`_.
> >   +11. Browse to the <topic-branch-name> branch on your forked Git repository and [open a Pull Request](http://help.github.com/send-pull-requests/). Give it a clear title and description.
> 
> Are you good with this then? Can I merge?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/jboss-jdf/jboss-as-quickstart/pull/462/files#r3232569
> .

## 

Jason Porter
http://en.gravatar.com/lightguardjp

**Comment by rafabene** [comment]
It is really much better.

**Comment by sgilda** [comment]
Merged.

**Pull request closed without merging**